flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/classification/SVM.scala

```diff
@@ -18,23 +18,20 @@
 
 package org.apache.flink.ml.classification
 
-import org.apache.flink.api.common.typeinfo.TypeInformation
-import org.apache.flink.ml.pipeline.{PredictOperation, FitOperation, PredictDataSetOperation,
-Predictor}
-
-import scala.collection.mutable.ArrayBuffer
-import scala.util.Random
-
+import breeze.linalg.{DenseVector => BreezeDenseVector, Vector => BreezeVector}
 import org.apache.flink.api.common.functions.RichMapFunction
 import org.apache.flink.api.scala._
 import org.apache.flink.configuration.Configuration
-import org.apache.flink.ml._
 import org.apache.flink.ml.common.FlinkMLTools.ModuloKeyPartitioner
-import org.apache.flink.ml.common._
-import org.apache.flink.ml.math.{DenseVector, Vector}
+import org.apache.flink.ml.common.{Parameter => FlinkParameter, _}
```

Comment:
I would like to preserve `Parameter` for Flink. How about renaming classes for PMML library?